news/views.py

```diff
@@ -599,3 +581,64 @@ def list_newsletters(request):
     active_newsletters = Newsletter.objects.filter(active=True)
     return render(request, 'news/newsletters.html',
                   {'newsletters': active_newsletters})
+
+
+def update_user_task(request, api_call_type, data=None, optin=True, sync=False):
```

Comment:
I didn't notice any difference between this and the version that used to be in utils; you just moved this to eliminate needing to import it at run time to avoid the circular import, right?